lib/Lexer.php

```diff
@@ -74,21 +74,19 @@ public function lex(string $docblock): Tokens
         );
 
         $tokens = [];
-        $prevChunk = null;
         foreach ($chunks as $chunk) {
             [ $value, $offset ] = $chunk;
             $tokens[] = new Token(
                 $offset,
-                $this->resolveType($value, $prevChunk),
+                $this->resolveType($value),
```

Comment:
`$prevChunk` was unused. Is it all right?

Reply:
yep